db/migrate/20161128193206_transform_evss_data.rb

```diff
@@ -0,0 +1,17 @@
+class TransformEvssData < ActiveRecord::Migration
+  def up
+    DisabilityClaim.all.each do |claim|
+      claim.data.deep_transform_keys!(&:underscore)
+      claim.list_data.deep_transform_keys!(&:underscore)
+      claim.save
+    end
+  end
+
+  def down
+    DisabilityClaim.all.each do |claim|
```

Comment:
This is OK for now since we have a small number of claims in the DB at a time due to the daily purge, but it is better to use `find_each` here to avoid loading all objects into memory at once